vulnerabilities/models.py

```diff
@@ -144,6 +148,10 @@ class VulnerabilityReference(models.Model):
 
     objects = BaseQuerySet.as_manager()
 
+    date_created = models.DateTimeField(auto_now_add=True, null=True)
```

Comment:
How could this ever be null? I would rather have it not null.

Reply:
This won't be null ever, because `auto_now_add` is True, I will add a test to show this migration's results